src/masoniteorm/exceptions.py

```diff
@@ -4,3 +4,7 @@ class DriverNotFound(Exception):
 
 class ModelNotFound(Exception):
     pass
+
+
+class HTTP404(Exception):
```

Comment:
@josephmancuso 
To do that create this HTTP exception level here. But smell not so good.

Reply:
That's fine. I like it